kitsune/wiki/views.py

```diff
@@ -1768,3 +1769,18 @@ def parse_accept_lang_header(lang_string):
     # Changed here to get the locale name only
     result = [k for k, v in result]
     return result
+
+
+def pocket_article(request, article_id=None, document_slug=None):
+    """Pocket articles migrated to SUMO are redirected to the new URL"""
+    try:
+        # If we migrated the document, let's get the doc
+        doc = Document.objects.get(locale=request.LANGUAGE_CODE, slug=document_slug)
```

Comment:
Does this fallback to the default locale if there's not a translation?

Reply:
Dropped the `locale=` bit - this should now be agnostic to locale since we are literally looking for a specific slug.  Really just being sure it exists, then letting wiki.document handle things.